plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -124,6 +124,11 @@ func (c *podSecurityPolicyPlugin) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// Ignore mirror pods, as they are run regardless of the admission decision.
+	if _, isMirrorPod := pod.Annotations[api.MirrorPodAnnotationKey]; isMirrorPod {
```

Comment:
Is it possible, that a user will add this annotation to a pod and bypass this check?

Reply:
Ok, I found the answer: https://github.com/kubernetes/kubernetes/issues/53467#issuecomment-334314148